src/common/util/copy-clipboard.ts

```diff
@@ -1,8 +1,12 @@
 export const copyToClipboard = (str) => {
-  const el = document.createElement("textarea");
-  el.value = str;
-  document.body.appendChild(el);
-  el.select();
-  document.execCommand("copy");
-  document.body.removeChild(el);
+  if (navigator.clipboard) {
+    navigator.clipboard.writeText(str);
```

Comment:
This returns a promise, we should await it and catch any errors so we can fallback